moveit_core/collision_detection/src/collision_matrix.cpp

```diff
@@ -213,19 +217,23 @@ void AllowedCollisionMatrix::setEntry(const std::string& name, bool allowed)
 {
   std::string last = name;
   for (auto& entry : entries_)
+  {
     if (name != entry.first && last != entry.first)
     {
       last = entry.first;
       setEntry(name, entry.first, allowed);
     }
+  }
 }
 
 void AllowedCollisionMatrix::setEntry(bool allowed)
 {
   const AllowedCollision::Type v = allowed ? AllowedCollision::ALWAYS : AllowedCollision::NEVER;
   for (auto& entry : entries_)
+  {
     for (auto& it2 : entry.second)
```

Comment:
I think you need to run it once more to add the braces to the inner for-loop

Reply:
Thanks for catching that.  I re-ran it, hopefully it got them all this time.